.clang-tidy.full

```diff
@@ -253,7 +253,7 @@ CheckOptions:
   - key:             modernize-pass-by-value.ValuesOnly
     value:           'false'
   - key:             readability-function-cognitive-complexity.IgnoreMacros
-    value:           'true'
+    value:           'false'
```

Comment:
Is that change intended? Don't we want to keep that value set to `true` so it works as on recent enough version of clang-tidy?

Reply:
This was not intended to be committed